base-internal/releases/node-22/22.15.1-glibc-2.31/build.sh

```diff
@@ -0,0 +1,7 @@
+set e+x
+
+# build image with Cypress dependencies
+LOCAL_NAME=cypress/base-internal:22.15.1-bullseye-python3.8-gcc-10.5
```

Comment:
ah I think the gcc was a previous name
```suggestion
LOCAL_NAME=cypress/base-internal:22.15.1-glibc-2.31
```